### PR TITLE
[#2659] Catalogue for Resource Profile 4.0

### LINKED ISSUE #2659: Add new field Catalogue to Dependecies  (Resource Profile v4.0)
Add new field Catalogue to Dependencies for Resource form
image
https://wiki.eoscfuture.eu/display/EOSCF/v4.00+EOSC+Resource+Profile

### PR DESCRIPTION
**Pull request by goreck888**
Add catalogue field in the dependencies section for Resource form
Closes #2659

### CODE REVIEW

**Comment by Marcelinna** [comment]
changelog

**Comment by Marcelinna** [comment]
Catalogue should be visible in dependencies in Details
image

File: app/views/backoffice/services/_form.html.haml [hunk 1]
```diff
@@ -289,7 +289,7 @@
         .collapse{ id: "dependencies", "aria-labelledby": "dependencies-header",
             "data-parent": "#accordion-form",
             class: ("show" unless collapsed?(service, [:required_services, :related_services,
-                                                      :related_platforms, :platforms])) }
+                                                      :related_platforms, :platforms, :catalogue])) }
```
Comment (kmarszalek) [inline app/views/backoffice/services/_form.html.haml:292]: should you add :catalogue at 278?

**Review by kmarszalek — CHANGES_REQUESTED** [review]
changelog and _form.haml.html at 278?
missing Resource view :)

**Review by kmarszalek — APPROVED** [review]
now its ok, thx for correcing my changelog typo

**Pull request merged**